src/coreclr/build-runtime.sh

```diff
@@ -21,7 +21,6 @@ export PYTHON
 usage_list+=("-nopgooptimize: do not use profile guided optimizations.")
 usage_list+=("-pgoinstrument: generate instrumented code for profile guided optimization enabled binaries.")
 usage_list+=("-skipcrossarchnative: Skip building cross-architecture native binaries.")
-usage_list+=("-skiprestoreoptdata: skip restoring optimization data.")
```

Comment:
@am11 this seems like an unintentional change?

Reply:
@akoeplinger, I called it out here https://github.com/dotnet/runtime/pull/34211#issuecomment-606255018. This option in help menu was removed from code, the case-esac block which handles options, and superseded by `-nopgooptimize`. For example, to quickly build coreclr native components with PAL tests on an unsupported Unix (e.g. SunOS, QNX), where `dontet(1)`/msbuild is currently unavailable, we can use:

```sh
./src/coreclr/build-runtime.sh -skipgenerateversion -nopgooptimize \
    -cmakeargs -DCLR_CMAKE_BUILD_TESTS=1
```

Reply:
Ok. It's still in src/coreclr/build-runtime.cmd that's why I was confused :)